pkgs/by-name/ie/ietf-cli/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  fetchFromGitHub,
+  python3,
+  rsync,
+  nix-update-script,
+}:
+python3.pkgs.buildPythonApplication rec {
+  name = "ietf-cli";
+  version = "1.27";
+  format = "other";
+
+  src = fetchFromGitHub {
+    owner = "paulehoffman";
+    repo = "ietf-cli";
+    rev = version;
```

Comment:
`tag = "v${version}"`

Reply:
done

Reply:
well, no, actually. I got a bit confused about my git state when 'fixing' this and I've taken a look at it again since the CI was failing.

The upstream author uses tags without a `v` prefix (see [here](https://github.com/paulehoffman/ietf-cli/tags)) so I've reverted it back to `tag = version`.

Reply:
Ah, sorry, I didn't actually check upstream, I assumed